Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridColumnHeadersPresenter.cs

```diff
@@ -359,13 +359,10 @@ protected override Size MeasureOverride(Size availableSize)
             // then we will resize all the columns to fit the available space.
             if (this.OwningGrid.UsesStarSizing && !this.OwningGrid.AutoSizingColumns)
             {
-                double adjustment = double.IsPositiveInfinity(availableSize.Width) ? this.OwningGrid.CellsWidth : availableSize.Width - totalDisplayWidth;
-                totalDisplayWidth += adjustment - this.OwningGrid.AdjustColumnWidths(0, adjustment, false);
```

Comment:
Same question here, this seems like it should still have an impact on the final calc?

Reply:
Yes, I will restore the call to `AdjustColumnWidths()` 

---
In reply to: [390620971](https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3155#discussion_r390620971) [](ancestors = 390620971)